doc/en/user/source/extensions/css/workshop/linestring.rst

```diff
@@ -0,0 +1,500 @@
+.. _extensions_css_cartography_linestring:
+
+Lines
+=====
+
+We will start our tour of CSS styling by looking at the representation of lines.
+
+.. figure:: img/LineSymbology.svg
+   
+   LineString Geometry
+
+Review of line symbology:
+
+* Lines are used to represent physical details that are too small to be represented at the current scale. Line work can also be used to model non-physical ideas such as network connectivity, or the boundary between land-use classifications. **The visual width of lines do not change depending on scale.**
+
+* Lines are recording as LineStrings or Curves depending on the geometry model used.
+
+* SLD uses a **LineSymbolizer** record how the shape of a line is drawn. The primary characteristic documented is the **Stroke** used to draw each segment between vertices.
+
+* Labeling of line work is anchored to the mid-point of the line. GeoServer provides a vendor option to allow label rotation aligned with line segments.
+
+For our exercises we are going to be using simple CSS documents, often consisting of a single rule, in order to focus on the properties used for line symbology.
+
+Each exercise makes use of the ``ne:roads`` layer.
```

Comment:
The prerequisites page is missing. We talked about an initial page showing what users will need to set this workshop up (for example, they don't have ``ne:roads``).

This could all go on the index page rather than a separate page if it's simpler. I guess it depends on how long the content ends up being.